ultra/install_deps.sh

```diff
@@ -74,9 +81,90 @@ function do_install_for_macos {
     fi
 }
 
+function do_install_for_WSL {
+    # We currently only support Ubuntu distribution for WSL (Windows subsystem
+    # for Linux). The main difference between WSL and native installation is that
+    # WSL does not support python3.7, instead python3.8 and its dependencies will be
+    # installed
```

Comment:
Sorry just saw this, is Python 3.8 an issue for 18.04 and 20.04 on WSL? It seems we are installing 3.7 on 18.04 for WSL.

Reply:
Resolved offline.